requirements_tracker.txt

```diff
@@ -0,0 +1,3 @@
+experiment_impact_tracker
+gpustat
```

Comment:
Should this be a separate `requirements_tracker.txt` file or do I need to add it to the main `requirements.txt` file?

Reply:
I'd be ok with adding this to the main requirements.txt file, especially if hardware resource usage tracking adds marginal overhead.

Curious about other people's opinions on this: @dantreiman @w4nderlust @tgaddair